server.js

```diff
@@ -24,7 +16,41 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Find Netflix data')
+})
+
+// Search the whole data base
+app.get('/movies', (req, res) => {
+  res.json(netflixData)
+})
+
+// Search on country and type Movie or TV Show
+app.get('/country/:country', (req, res) => {
+  const country = req.params.country
+  const showType = req.query.type
+  let movieFromCountry = netflixData.filter((item) => item.country === country)
+  
+  if (showType) {
+    movieFromCountry = movieFromCountry.filter((item) => item.type ===  showType)
+  }
+
+  res.json(movieFromCountry)
+})
+
+// Query search on Titel always toLowerCase
+app.get('/search', (req, res) => {
```

Comment:
I think this would be nice if it were in the `/movies` endpoint, too. As a rule of thumb, you only need to make new endpoints when it will return a different type of data, so a list of movies on `/movies`, a single movie on `/movies/:id`, movies for a country on `/movies?country=sweden`, or searching for a movie on `/movies?title=Jumanji`. If an endpoint will perform an action such as creating, updating or deleting an entity, it'll be a new endpoint, too.